OpenIPSL/Electrical/Solar/PowerFactory/DIgSILENT/PV_Plant.mo

```diff
@@ -0,0 +1,144 @@
+    Placement(visible = true, transformation(origin = {-50, 10}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+equation
+  connect(generator.p, p) annotation(
+    Line(points = {{82, 0}, {110, 0}}, color = {0, 0, 255}));
+  connect(pu_to_W.u, generator.P) annotation(
+    Line(points = {{82, -30}, {90, -30}, {90, -4}, {82, -4}}, color = {0, 0, 127}));
+  connect(controller.id_ref, generator.id_ref) annotation(
+    Line(points = {{42, 8}, {62, 8}}, color = {0, 0, 127}));
+  connect(controller.iq_ref, generator.iq_ref) annotation(
+    Line(points = {{42, 0}, {50, 0}, {50, 4}, {62, 4}}, color = {0, 0, 127}));
+  connect(controller.uac, generator.v) annotation(
+    Line(points = {{22, 0}, {16, 0}, {16, 20}, {86, 20}, {86, 8}, {82, 8}}, color = {0, 0, 127}));
+  connect(not_implemented_FRT.y, controller.pred) annotation(
+    Line(points = {{1, -50}, {14, -50}, {14, -4}, {22, -4}}, color = {0, 0, 127}));
+  connect(busbar.I_pv, pv_array.Iarray) annotation(
+    Line(points = {{-19, 16}, {-39, 16}}, color = {0, 0, 127}));
+  connect(pu_to_W.y, busbar.P_conv) annotation(
+    Line(points = {{60, -30}, {-24, -30}, {-24, 8}, {-19, 8}}, color = {0, 0, 127}));
+  connect(busbar.Udc, controller.vdcin) annotation(
+    Line(points = {{2, 12}, {22, 12}}, color = {0, 0, 127}));
+  connect(busbar.Udc, pv_array.Uarray) annotation(
+    Line(points = {{2, 12}, {6, 12}, {6, 26}, {-64, 26}, {-64, 18}, {-58, 18}}, color = {0, 0, 127}));
+  connect(pv_array.Vmpp_array, controller.vdcref) annotation(
+    Line(points = {{-38, 4}, {-30, 4}, {-30, 0}, {6, 0}, {6, 8}, {22, 8}}, color = {0, 0, 127}));
+protected
+  annotation(
+    Diagram(coordinateSystem(extent = {{-100, -100}, {100, 100}})),
+    Icon(graphics = {Rectangle(lineColor = {118, 18, 62},fillColor = {255, 255, 255}, fillPattern = FillPattern.Solid, extent = {{-100, 100}, {100, -100}}), Line(origin = {0, 60.31}, points = {{-100, 39.6877}, {0, -40.3123}, {100, 39.6877}, {100, 39.6877}}, color = {118, 18, 62}), Text(origin = {0, -10}, lineColor = {0, 0, 255}, extent = {{-100, 150}, {100, 110}}, textString = "%name")}),
+        Documentation(info = "<html>
+<p>
+A PV Plant implemented according to the DIgSILENT template in PowerFactory. 
+</p>
+<p>NOTE 1: The PLL Dynamics are missing along with the active power reduction models for FRT events.
+</p>
+<p>NOTE 2: Keep in mind that if external irradiance is connected it needs to be matched for the inital active power for steady-state initialization.
+</p>
+</html>", revisions = "<html>
+<table cellspacing=\"1\" cellpadding=\"1\" border=\"1\"><tr>
+<td><p>Model Name</p></td>
+<td><p>DIgSILENT PV System 3PH</p></td>
+</tr>
+<tr>
+<td><p>Reference</p></td>
+<td><p>PowerFactory Implementation in \DIgSILENT Library\Templates\Photovoltaic</p></td>
```

Comment:
I had a warning when loaded the package in OMEdit basically because of this \D, \T and \P. It says that this is not valid in Modelica and that it would interpret it as \\. Therefore I believe it should be: 

```suggestion
<td><p>PowerFactory Implementation in \\DIgSILENT Library\\Templates\\Photovoltaic</p></td>
```

Reply:
Forward slashes are universal (Win and POSIX), backslashes are not. So I simply changed this to `/` instead.